tools/docker-compose/bootstrap_development.sh

```diff
@@ -34,3 +34,4 @@ make init
 
 mkdir -p /awx_devel/awx/public/static
 mkdir -p /awx_devel/awx/ui/static
+mkdir -p /awx_devel/awx/job_status
```

Comment:
Prior to this PR, there's code that this creates this directory (if it doesn't exist) in `tasks.py`:

https://github.com/ansible/awx/pull/833/files#diff-9d4ea1dd908b35fb92eaede4bd10bb46L743

That code's gone now, and I think it makes more sense to just ensure this directory exists in the dev env at container start time.